scripts/iftop.sh

```diff
@@ -13,4 +13,4 @@ if sudo true; then
 sudo="sudo -n"
 fi
 
-$sudo iftop -i "$(ifconfig | grep mtu | grep -iv loopback | grep -i running | awk 'BEGIN { FS = ":" } ; {print $1}')" -nNbBP  -t
+exec "$sudo" iftop -i "$(ifconfig | grep mtu | grep -iv loopback | grep -i running | awk 'BEGIN { FS = ":" } ; {print $1}')" -nNbBP -t -L 1000
```

Comment:
I think this'll cause `sudo -n` to be executed which'll fail because there's no program called `sudo -n`.

`exec $sudo ...` is what we want here, and suppress the bogus shellcheck complaint